srcpkgs/libtar/template

```diff
@@ -1,15 +1,16 @@
 # Template file for 'libtar'
 pkgname=libtar
 version=1.2.20
-revision=2
+revision=3
 wrksrc="$pkgname"
+patch_args="-Np1"
 build_style=gnu-configure
 hostmakedepends="automake libtool pkg-config"
 makedepends="zlib-devel"
 short_desc="C library for manipulating POSIX tar files"
 maintainer="Juan RP <xtraeme@voidlinux.eu>"
-license="BSD"
-homepage="http://www.feep.net/libtar/"
+license="BSD-3-Clause"
+homepage="https://repo.or.cz/w/libtar.git"
 distfiles="ftp://ftp.netbsd.org/pub/pkgsrc/distfiles/libtar-${version}.tar.gz"
```

Comment:
If possible switch to https:// or http://

ftp:// is know to cause problems with travis